src/test/java/emissary/test/core/junit5/RegressionTest.java

```diff
@@ -51,14 +53,14 @@ public abstract class RegressionTest extends ExtractionTest {
     private List<SimplifiedLogEvent> actualSimplifiedLogEvents;
 
     /**
-     * Override this to generate XML for data files.
+     * Override this or set the generateAnswers system property to true to generate XML for data files.
      * 
      * @return defaults to false if no XML should be generated (i.e. normal case of executing tests) or true to generate
      *         automatically
      */
     @ForOverride
     protected boolean generateAnswers() {
-        return false;
+        return Boolean.getBoolean("generateAnswers");
```

Comment:
The only concern I have is that any existing tests that already override this method won't pick up this behavior, and it might not be obvious why,